src/locale/locales/pt/messages.

```diff
@@ -0,0 +1,2468 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-12-22 01:56+0530\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
```

Comment:
Please run `yarn intl:extract` and `yarn intl:compile` to get the final compilation of the `messages.po` file

Reply:
I compiled it in .po but when creating the file here because I can't upload it, I forgot about the .po done now!